kuma/wiki/models.py

```diff
@@ -1805,6 +1756,14 @@ def get_section_content(self, section_id):
         """Convenience method to extract the content for a single section"""
         return self.document.extract_section(self.content, section_id)
 
+    def get_tidied_content(self):
+        if self.tidied_content:
+            return self.tidied_content
+        else:
+            from .tasks import tidy_revision_content
+            tidy_revision_content.delay(self.pk)
+            return tidy_content(self.content)
```

Comment:
Could a crawler still request a `compare_revisions` view for some old revisions and trigger this CPU-intensive tidying of content with its first request?